sdk/storage/Azure.Storage.Blobs/src/BlobUriBuilder.cs

```diff
@@ -161,6 +164,20 @@ public string Query
         }
         private string _query;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BlobUriBuilder"/>
+        /// class for the specified Azure Storage Account.
+        /// </summary>
+        /// <param name="accountName">
+        /// The account name of the target storage account.
+        /// </param>
+        /// <param name="storageSuffix">
+        /// The suffix of the storage cloud. Defaults to Azure public cloud (blob.core.windows.net).
+        /// </param>
+        public BlobUriBuilder(string accountName, string storageSuffix = "blob.core.windows.net") : this(new Uri($"https://{accountName}.{storageSuffix.TrimStart('.')}"))
```

Comment:
Seeing as my other comment got marked as resolved since the source changed.

Accepting only the storage account name as a default, and assuming the ending suffix (e.g. blob.core.windows.net) is not a good idea.

We should be taking in the ending suffix of the endpoint. This would not be a good experience for customers who are new to storage and have a different type of endpoint details.

Reply:
this code _does_ take in the suffix of the endpoint...